docs/design/2018-10-24-view-support.md

```diff
@@ -0,0 +1,49 @@
+# Proposal: Implement View Feature
+
+- Author(s):     AndrewDi
+- Last updated:  2018-10-24
+- Discussion at: https://github.com/pingcap/tidb/issues/7974
+
+## Abstract
+This proposal proposes to implement view feature in TiDB, aimed to make sql more easy to write.
+
+## Background
+A database view is a searchable object in a database that is defined by a query.  Though a view doesn’t store data, some refer to a views as “virtual tables,” you can query a view like you can a table.  A view can combine data from two or more table, using joins, and also just contain a subset of information.  This makes them convenient to abstract, or hide, complicated queries.
+
+Below is a visual depiction of a view:
+  ![AnatomyOfAview](./view.png)
```

Comment:
Now you can move it to `imgs` subdirectory

Reply:
fine